webpack/assets/javascripts/react_app/routes/ExternalRoutes.js

```diff
@@ -0,0 +1,25 @@
+import { Route } from 'react-router-dom';
+import React from 'react';
+import { addGlobalFill } from '../components/common/Fill/GlobalFill';
+import { renderRoute } from './RoutingService';
+
+/**
+ * Adds a plugin's routes into core
+ * @param  {String} id  plugin's id - can be its name
+ * @param  {Array}   routes an array that contains a plugin's routes
+ */
+export const addGlobalRoutes = (id, routes) =>
```

Comment:
Should we rename it to `registerRoute` to stay aligned with 
`registerReducer` and `componentRegisty.register` that we provide to plugins?

Reply:
I like it :+1: